frontend/src/storage/cache.ts

```diff
@@ -0,0 +1,19 @@
+import { InMemoryCache } from "@apollo/client";
+
+import store from "./store";
+
+const cache: InMemoryCache = new InMemoryCache({
+  typePolicies: {
+    Query: {
+      fields: {
+        store: {
```

Comment:
I think it's _okay_ to have a single InMemoryCache field for `store`, but I think that one of the most powerful features of Apollo's reactive variables is the ability to automatically update previously-made Queries based on a cached variable having been updated.

Taking advantage of this capability would require that we cache those query variables individually. I understand that we're not using it that way, here, but just wanted to call it out